### PR TITLE
chore(core): reduce release size by 2MB

### PR DESCRIPTION
**Pull request by bluestreak01**
- reduced release size by not double-packaging the Web Console
- made JavaDoc generator only run during maven central release. Otherwise it delays docker build by 10 min at least
- reduced compilation warnings by consolidating Java proprietary API imports in `Unsafe` class
- fixed `questdb.exe`. Somehow we checked in binary, which referenced `ServerMain` as `Server`, hence not working on Windows

Oh, no, I committed to a wrong branch :(

So this commit https://github.com/questdb/questdb/pull/2557/commits/633b25dab0c236aa2b37008339c295713311e721 addresses @bziobrowski question https://github.com/questdb/questdb/pull/2423#issuecomment-1257922359

The solution is:
- remove cast `char` to timestamp. It makes no sense. This lead to `cast('2' as timestamp) produce something bizarre.
- leave `cast(str as timestamp)` to use timestamp-floor function
- change `to_timestamp(S)` to always interpret arg as micros. Similar function in PG interprets string as double, where scale values used for millis or micros and integers as seconds. I feel our approach is clearer

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/columns/ColumnUtils.java|3|3|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/columns/ColumnUtils.java|3|3|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 59 / 65 (90.77%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/date/ToTimestampFunctionFactory.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/FunctionParser.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/cairo/ImplicitCastException.java|18|21|85.71%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/cast/CastGeoHashToGeoHashFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/WriterRowUtils.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/cast/CastStrToTimestampFunctionFactory.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/bind/BindVariableServiceImpl.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/functions/columns/ColumnUtils.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlUtil.java|25|25|100.00%|

File: core/src/test/java/io/questdb/cairo/GeoHashesTest.java [hunk 1]
```diff
@@ -697,9 +696,8 @@ private void testUnsafeFromStringTruncatingNl(CharSequence token, BiFunction<Lon
         final long lo = Unsafe.malloc(len, MemoryTag.NATIVE_DEFAULT);
         final long hi = lo + len;
         try {
-            sun.misc.Unsafe unsafe = Unsafe.getUnsafe();
             for (long p = lo; p < hi; p++) {
-                unsafe.putByte(p, (byte) token.charAt((int) (p - lo)));
+                Unsafe.getUnsafe().putByte(p, (byte) token.charAt((int) (p - lo)));
```
Comment (marregui) [inline core/src/test/java/io/questdb/cairo/GeoHashesTest.java:700]: why access it in the loop and not as before, I am genuinely interested, there must be a reason I fail to see.

**Review by marregui — APPROVED** [review]
LGTM

File: core/src/main/java/io/questdb/griffin/engine/functions/columns/ColumnUtils.java [hunk 2]
```diff
@@ -30,13 +30,12 @@ public class ColumnUtils {
     static final int STATIC_COLUMN_COUNT = 32;
 
     public static void symbolColumnUpdateKeys(long columnMemory, long columnMemorySize, long remapTableMemory, long remapMemorySize) {
-        final sun.misc.Unsafe unsafe = Unsafe.getUnsafe();
         for (int offset = 0; offset < columnMemorySize; offset += Integer.BYTES) {
-            final int oldKey = unsafe.getInt(columnMemory + offset);
+            final int oldKey = Unsafe.getUnsafe().getInt(columnMemory + offset);
```
[the remark this answers is not in the document]
Comment (bluestreak01) [reply core/src/main/java/io/questdb/griffin/engine/functions/columns/ColumnUtils.java:34]: JIT will deal with this. I had micro-benched this before and there is no effect on performance.

Creating "unsafe" variable in a class however does trigger more warnings on compilation, e.g. more classes use "proprietary" API. I wanted to have all proprietary API access from single place if possible

**Pull request merged**